cpp-package/example/README.md

```diff
@@ -0,0 +1,112 @@
+# MXNet C++ Package Examples
+
+## Building C++ examples
+
+The examples are built while building the MXNet library and cpp-package from source . However, they can be built manually as follows
```

Comment:
Really? I had to build them manually.
...from source. However,

Reply:
Yes. I noticed it when I built it on mac and under ubuntu docker. The executables are kept under ./build/cpp-package/example directory.